Sprint-3/3-mandatory-practice/implement/get-ordinal-number.js

```diff
@@ -1,5 +1,18 @@
 function getOrdinalNumber(num) {
- return "1st";
+  if (num % 100 >= 11 && num % 100 <= 13) {
+    return num + "th";
+  }
+
+  switch (num % 10) {
+    case 1:
+      return num + "st";
+    case 2:
+      return num + "nd";
+    case 3:
+      return num + "rd";
+    default:
+      return num + "th";
```

Comment:
using switch case is a good way to handle  check different conditions instead of  using if and if else